### ISSUE TITLE
prereqs.sh attempts to install epel-release along with packages inside epel repository

### ISSUE THREAD

**Issue by TrevorBenson** [cardano-community/guild-operators#482]
**Describe the bug**
prereqs.sh does not succeed if epel-release is not already installed. The test it suggests (missing the word install) says to do: 

```Using yum to prepare packages for "CentOS Linux" system
  Updating system packages...
  Installing missing prerequisite packages, if any..
Error: Unable to find a match: libsodium-devel srm
An error occurred while installing the prerequisite packages, please investigate by using the command below:
sudo yum -y epel-release python3 coreutils pkgconfig libffi-devel gmp-devel openssl-devel ncurses-libs ncurses-compat-libs systemd systemd-devel libsodium-devel zlib-devel make gcc-c++ tmux git wget jq gnupg libtool autoconf srm iproute bc tcptraceroute
It would be best if you could submit an issue at https://github.com/cardano-community/guild-operators with the details to tackle in future, as some errors may be due to external/already present dependencies
```

TYPO:
It should be suggesting `sudo yum -y install ....`

BUG:
You cannot request an installation of epel-release along with packages that exist inside epel repository as there is no metadata to query it fails to find packages for libsodium-devel and srm if epel-release is not installed before prereqs.sh is run.

BUG:
(let me know if you prefer this to go into its own issue, I will likely open one anyway but wanted to add this note here anyway)
Fedora is not an enterprise linux distribution and therefore does not have an "epel-release" package to install, packages from epel-release are available in other Fedora repositories since it is the "bleeding edge" distribution that the enterprise distro is derived from.

**To Reproduce**
1. Run prereqs.sh on CentOS/RHEL distributions
2. See error `Error: Unable to find a match: libsodium-devel srm`

**Expected behavior**

1. CentOS/RHEL distros should first install epel-release and then ask for packages that that exist inside epel repository: libsodium-devel srm
2. Fedora distros should not request epel-release as it is not required and only for enterprise distributions.

**Version:**
 - OS: CentOS 8
 - Product version: CNTools 5.4.1
 - Cardano Node version: N/A
 - Network you're connecting to: N/A/genesis.json`]

**Comment by rdlrt** [comment]
Agreed! will add that

**Comment by rdlrt** [comment]
Closed in #482

**Comment by TrevorBenson** [comment]
> Agreed! will add the first point
> 
> About your second point, epel-release is [only added ](https://github.com/cardano-community/guild-operators/blob/master/scripts/cnode-helper-scripts/prereqs.sh#L138) if release is not Fedora

Ahh, I had quickly glanced at prereqs.sh and totally missed the ! operator at the beginning, my bad!
